cmd.js

```diff
@@ -5,27 +5,30 @@ var commandline={
     run:runCommand
 };
 
-function runCommand(command){
+function runCommand(command,options){
     //return refrence to the child process
     return exec(
-        command
+        command,
+        options || {}
     );
 }
 
-function getString(command,callback){
+function getString(command,options,callback){
```

Comment:
this is a breaking change. I would add this in if you moved it to the end, instead of in the middle.

Reply:
it should be in the middle, following exec function

Reply:
In most of the functions of nodejs, the callback parameter is the last.